### PR TITLE
Add mouse seed database

### PR DESCRIPTION
**Pull request by oplantalech**
A specific seed database for mouse has been developed. This is only the part with no pdb tables, and it works fine with the schema 2.1.0 and the part with pdb tables that it is already available for human in PR #21

### CODE REVIEW

**Comment by pieterlukasse** [comment]
@oplantalech can you please also add the information on how the DB was made? Then the steps can be repeated again once we need a new version.

**Comment by pieterlukasse** [comment]
maybe we could make a subfolder for mouse, like `seedDB/mouse/` or just another folder called `seedDB_mouse`

**Comment by oplantalech** [comment]
@pieterlukasse Done.

**Comment by Sjoerd-van-Hagen** [comment]
@n1zea144 could you merge this PR, please?

**Comment by oplantalech** [comment]
@Sjoerd-van-Hagen @n1zea144 Now the PR is ready to merge

**Comment by pieterlukasse** [comment]
@oplantalech regarding my first comment: where was this done?

**Comment by pieterlukasse** [comment]
never mind, found it. :+1:

**Comment by pieterlukasse** [comment]
:warning: one remark: this one needs to be merged first so that the documentation link you added works: https://github.com/cBioPortal/datahub/pull/21

**Comment by oplantalech** [comment]
@pieterlukasse Agree. Once PR #21 is merged, I will remove the do not merge tag from this PR. Apart from that, this PR is ready to merge.

**Comment by sandertan** [comment]
I tested this, and everything looks good! Nice that https://github.com/cBioPortal/cbioportal/pull/2210 was merged quickly, which was necessary for this PR.

Ready to merge.

**Comment by sandertan** [comment]
Ready to merge when cBioPortal 1.5.0 and human seed-db (https://github.com/cBioPortal/datahub/pull/21) are released.

**Comment by oplantalech** [comment]
I updated the schema to 2.1.0

**Comment by oplantalech** [comment]
@n1zea144 This PR is also ready to merge.

File: seedDB_mouse/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+These files are MySQL database dump for seeding a new instance of the cBioPortal DB for mouse. They contain all the necessary background data for a properly functioning cBioPortal website, including cancer types, gene, uniprot-mappings, drug, pdb and network data. Data is compatible with:
+
```
Comment (pieterlukasse) [inline seedDB_mouse/README.md]: pdb is not included here

File: seedDB_mouse/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+- DB schema version **2.1.0**
+
+Which is suitable for:
+
+- release version **1.5.0** of cBioPortal.
```
Comment (pieterlukasse) [inline seedDB_mouse/README.md]: 1.5.1

**Pull request merged**